CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 * Fixed an issue where changes made to `NavigationViewController.showsReportFeedback`, `NavigationViewController.showsSpeedLimits`, `NavigationViewController.detailedFeedbackEnabled`, `NavigationViewController.floatingButtonsPosition` and `NavigationViewController.floatingButtons` before `NavigationViewController` presentation were not saved. ([#3718](https://github.com/mapbox/mapbox-navigation-ios/pull/3718))
 * Fixed an issue where `SpeechSynthesizing.managesAudioSession` was ignored by `RouteVoiceController`. ([#3572](https://github.com/mapbox/mapbox-navigation-ios/pull/3572))
 * Location tracking and routing resources are now freed if no `RouteController` or `PassiveLocationManager` instance is actively being used. ([#3724](https://github.com/mapbox/mapbox-navigation-ios/pull/3724))
+* Added tileset profile configuration for Free Drive, Active Guidance, `MapboRoutingProvider` and Tileset Descriptors API. Now, `RouteOptions.profileIdentifier` is used to determine which type of tiles should be used for navigation. Please make sure to pay attention to `MapboxRoutingProfider` and `TilesetDescriptorFactory` arguments when configuring `ProfileIdentifier`. ([#3717](https://github.com/mapbox/mapbox-navigation-ios/pull/3717))
```

Comment:
Free Drive, Active Guidance and Tileset Descriptors API aren’t terms we use anywhere else in the documentation, at least not as formal, capitalized names of APIs that look like server APIs. Let’s refer to specific classes or methods that the developer can use. Something like this (but please fact-check the specific symbols I named):

```suggestion
* Added an optional `datasetProfile` argument to the `MapboxRoutingProvider(_:settings:datasetProfile:)`, `PassiveLocationManager(directions:systemLocationManager:eventsManagerType:userInfo:datasetProfile:),` `TilesetDescriptorFactory.getSpecificVersion(version:completionQueue:datasetProfile:completion:)`, and `TilesetDescriptorFactory.getLatest(completionQueue:datasetProfile:completion:)` methods for obtaining routing tiles optimized for a particular mode of transportation. Make sure to configure `MapboxRoutingProvider` and `TilesetDescriptorFactory` with the correct dataset profile if you customize `Directions.profileIdentifier`. ([#3717](https://github.com/mapbox/mapbox-navigation-ios/pull/3717))
```